docs/rule/require-form-method.md

```diff
@@ -0,0 +1,40 @@
+# require-form-method
+
+This rule requires all `<form>` elements to have `method` attribute with `POST` or `GET` value.
+
+By default `form` elements without `method` attribute are submitted as `GET` requests.
+In usual applications `submit` event listeners are attached to `form` elements and `event.preventDefault()` is called to avoid form submission.
+
+However in case of failure to prevent default action, form submission as `GET` request can leak sensitive end-user information.
+Examples of valid uses for `GET` requests are non-secure data, bookmarking the submission result, data search query strings.
```

Comment:
I would prefer bullet-point list instead of a sentence here to make it easier to parse the acceptable usages.

Reply:
Converted to list.